blog/_posts/2017-07-19-Team-Serv0101.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Meet Us - Team Serv0101 
+layout: post
+created_at: Wed Jul 19 2017
+permalink: blog/2017-07-19-Meet Us - Team Serv0101
+current: blog
+author: Neha & Rakhi
+categories:
+- student-posts
+---
+
+## Introducing Team Serv0101
+
+<img src="/img/blog/2017/TeamServo_Neha_Rakhi.jpg" alt="Neha and Rakhi">
```

Comment:
please add image credits to all images as explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images). thanks!